holmes/core/toolset_manager.py

```
@@ -128,19 +135,33 @@
+                future = executor.submit(
+                    toolset.check_prerequisites, quiet=self.suppress_logging
+                )
+                futures[future] = toolset
+                active_checks.append(toolset.name)
 
-            for _ in concurrent.futures.as_completed(futures):
-                pass
+            completed = 0
+            total = len(futures)
+            for future in concurrent.futures.as_completed(futures):
+                completed += 1
```

Ruff (0.12.2)

Use `enumerate()` for index variable `completed` in `for` loop

(SIM113)